dashboard/client/src/type/serve.ts

```diff
@@ -0,0 +1,64 @@
+export enum ServeApplicationStatus {
+  // Keep in sync with ApplicationStatus in python/ray/serve/_private/common.py
+  NOT_STARTED = "NOT_STARTED",
+  DEPLOYING = "DEPLOYING",
+  RUNNING = "RUNNING",
+  DEPLOY_FAILED = "DEPLOY_FAILED",
+  DELETING = "DELETING",
+}
+
+export type ServeApplication = {
+  name: string;
+  route_prefix: string;
+  docs_path: string | null;
+  status: ServeApplicationStatus;
+  message: string;
+  last_deployed_time_s: number;
+  deployed_app_config: Record<string, any>;
+  deployments: {
+    [name: string]: ServeDeployment;
+  };
+};
+
+export enum ServeDeploymentStatus {
+  // Keep in sync with DeploymentStatus in python/ray/serve/_private/common.py
```

Comment:
Could you add a similar comment linking back to this file in [the `DeploymentStatus` class](https://github.com/ray-project/ray/blob/514bb3abeff8172677f35158746376e8cb4786ca/python/ray/serve/_private/common.py#L31)?